Formula/wxlua.rb

```diff
@@ -0,0 +1,47 @@
+class Wxlua < Formula
+  desc "Lua bindings for wxWidgets cross-platform GUI toolkit"
+  homepage "https://github.com/pkulchenko/wxlua"
+  url "https://github.com/pkulchenko/wxlua/archive/refs/tags/v3.2.0.2.tar.gz"
+  sha256 "62abe571803a9748e19e86e39cb0e254fd90a5925dc5f0e35669e693cbdb129e"
+  license "LGPL-2.0-or-later" => { with: "WxWindows-exception-3.1" }
+  head "https://github.com/pkulchenko/wxlua.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "lua"
+  depends_on "wxwidgets"
+
+  def install
+    lua = Formula["lua"]
+
+    args = std_cmake_args + %W[
+      -DwxLua_LUA_LIBRARY_VERSION=#{lua.version.major_minor}
+      -DwxLua_LUA_INCLUDE_DIR=#{lua.opt_include}/lua
+      -DwxLua_LUA_LIBRARY=#{lua.opt_lib/shared_library("liblua")}
+      -DwxLua_LUA_LIBRARY_USE_BUILTIN=FALSE
+      -DwxWidgets_COMPONENTS=gl;xrc;xml;net;media;propgrid;aui;stc;html;adv;core;base
+      -DwxLuaBind_COMPONENTS=gl;xrc;xml;net;media;propgrid;aui;stc;html;adv;core;base
+    ]
+
+    cd "wxLua" do
+      system "cmake", "-S", ".", "-B", "build-wxlua", *args
+
+      system "cmake", "--build", "build-wxlua"
+      system "cmake", "--install", "build-wxlua"
+    end
```

Comment:
```suggestion
    args = %W[
      -DwxLua_LUA_LIBRARY_VERSION=#{lua.version.major_minor}
      -DwxLua_LUA_INCLUDE_DIR=#{lua.opt_include}/lua
      -DwxLua_LUA_LIBRARY=#{lua.opt_lib/shared_library("liblua")}
      -DwxLua_LUA_LIBRARY_USE_BUILTIN=FALSE
      -DwxWidgets_COMPONENTS=gl;xrc;xml;net;media;propgrid;aui;stc;html;adv;core;base
      -DwxLuaBind_COMPONENTS=gl;xrc;xml;net;media;propgrid;aui;stc;html;adv;core;base
    ]

    system "cmake", "-S", "wxLua", "-B", "build-wxlua", *args, *std_cmake_args
    system "cmake", "--build", "build-wxlua"
    system "cmake", "--install", "build-wxlua"
```
Do we need to specify `*_COMPONENTS`?

Reply:
We do, because the default includes `richtext` which is a 40,000 line file which takes over 10 minutes to compile with optimizations. Other packages, such as the Arch Linux wxLua package, also disable the `richtext` component.